Find a functionally equivalent code:android.content.res.Resources.Theme.setTo:COMMENT
Method Modifier: public      
Comment:/**
 * Set this theme to hold the same contents as the theme
 * <var>other</var>.  If both of these themes are from the same
 * Resources object, they will be identical after this function
 * returns.  If they are from different Resources, only the resources
 * they have in common will be set in this theme.
 *
 * @param other The existing Theme to copy from.
 */

@@ -1,3 +1,5 @@
 {
     AssetManager.copyTheme(mTheme, other.mTheme);
+    mThemeResId = other.mThemeResId;
+    mKey = other.mKey;
 }

